client/src/components/OverviewPage/OverviewSeatInfo/OverviewSeatInfo.jsx

```diff
@@ -1,10 +1,10 @@
 import React, { useState, useEffect } from "react";
 import { observer } from "mobx-react-lite";
-import bookingStore from "../../stores/BookingStore";
-import Booking, { DeleteBookingRequest } from "../../domain/booking";
-import { useAuthContext } from "../../api/useAuthContext";
+import bookingStore from "../../../state/stores/BookingStore";
+import Booking, { DeleteBookingRequest } from "../../../models/booking";
+import { useAuthContext } from "../../../core/auth/useAuthContext";
 
-const SeatInfoModal = observer(({ onClose, selectedSeatId }) => {
+const OverviewSeatInfo = observer(({ onClose, selectedSeatId }) => {
```

Comment:
bedre å kalle det for BookingConfirmationModel, og plassere det på Common, da blir det litt mer generisk også kan gjenbruke i Admin siden